config/initializers/devise.rb

```diff
@@ -196,7 +196,7 @@ def skip_format?
   # Email regex used to validate email formats. It simply asserts that
   # one (and only one) @ exists in the given string. This is mainly
   # to give user feedback and not to assert the e-mail validity.
-  config.email_regexp = /\A[^@\s]+@[^@\s]+\z/
+  config.email_regexp = /(?=\A.{6,255}\z)(\A([\p{L}0-9]+[\w|\-|\.|\+]*)@((?i-mx:[\p{L}0-9]+([\-\.]{1}[\p{L}0-9]+)*\.[\p{L}]{2,63}))\z)/
```

Comment:
Since emails are going to come from GitHub, we don't have to worry much about validating them since GitHub will have already done that. I think we can leave this as-is.

Reply:
But users can change email from this screen. We should disable the functionality to change the email from this screen @excid3. What do you think?

<img width="1440" alt="Screenshot 2022-09-08 at 10 12 57 AM" src="https://user-images.githubusercontent.com/39980734/189036043-0a815af8-374b-4913-95f8-346783f1c654.png">

Reply:
Yeah, we can remove that since we'll want to use their email from GitHub always. 👍

Reply:
Ok, creating a PR for that.

Reply:
@excid3 Added PR #5 to close this one.